README.md

```diff
@@ -117,6 +70,27 @@ The output file is a JSON file in the format:
 We don't recommend using the `abbreviated` feature.
 It will omit entries that have fewer than a hard-coded number of projects that depend on them.
 
+### More details (possibly outdated)
+
+Mariner can be called from Javascript or from Typescript. You can see an example here:
+<https://github.com/indeedeng/Mariner/blob/master/examples/runOldCode.ts>
+
+Mariner is in transition from the old way of accessing GitHub data (REST) to the new way (GraphQL)
+
+To invoke mariner using the new GraphQL code, Invoke the finder(), passing the
+appropiate parameters in finder.findIssues() you can see an example here:
+<https://github.com/indeedeng/Mariner/blob/master/examples/runFasterCode.ts>
+
+If you are using the `examples/runOldCode.ts file`, (using the old REST code that is very slow)
+invoke the DependencyDetailsRetriever.run() method, passing appropriate parameters. Please
+see the [examples/runOldCode.ts](https://github.com/indeedeng/Mariner/blob/master/examples/runOldCode.ts) file for more information.
+
+For both the `runOldCode.ts` and `runFasterCode.ts` files you must create a token.
```

Comment:
Ooh. This section needs to be outside the "More Details" section. Probably a new subsection below Output File, named "GitHub Token". It should be ### level, as should the input and output file format sections. 

Instead of "For both the...files", let's just say "To run Mariner, you must..."